rllib/train.py

```diff
@@ -273,19 +266,11 @@ def patch_path(path):
             local_mode=args.local_mode,
         )
 
-    if IS_NOTEBOOK:
-        progress_reporter = JupyterNotebookReporter(
-            overwrite=verbose >= 3, print_intermediate_tables=verbose >= 1
-        )
-    else:
-        progress_reporter = CLIReporter(print_intermediate_tables=verbose >= 1)
```

Comment:
I don't fully understand why we can get rid of the `CLIReporter` here, as I'm not familiar with that reporter. But I trust that this case is covered.

Reply:
Instead of duplicating logic here, we defer to tune run which is being called inside `run_experiments`. There, the progress reporter is detected.

This means this is merely removal of unnecessary, duplicated code without any changes to logic.